cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -102,7 +102,7 @@ template <
     typename                                            PrivatizedDecodeOpT,            ///< The transform operator type for determining privatized counter indices from samples, one for each channel
     typename                                            OutputDecodeOpT,                ///< The transform operator type for determining output bin-ids from privatized counter indices, one for each channel
     typename                                            OffsetT>                        ///< Signed integer type for global offsets
-__launch_bounds__ (int(AgentHistogramPolicyT::BLOCK_THREADS))
+CUB_DETAIL_LAUNCH_BOUNDS(int(AgentHistogramPolicyT::BLOCK_THREADS))
```

Comment:
Should we move the cast to `int` into the macro?

Reply:
I think this would be a bit complected. I'd prefer the project to be refactored so that `BLOCK_THREADS` is `int` :)

Reply:
Yeah, that would be much better